### ISSUE TITLE
Doesn't recognize (some) common German nouns

### ISSUE THREAD

**Issue by SparkyCola** [xpheres/analyticaltranslator#1]
Hi, I just found your project on F-Droid and was wondering why it didn't recognise any noun (I checked "Tag" and "Regen" in PHP-Interface).

**Comment by xpheres** [comment]
Hello, thank you for your interest in this translator. I wish I can push the project forward but the ammount of work is huge... I wrote the vocabulary database myself word by word and there are still lots of work to do. You can check the vocabulary database yourself here:

http://lingoworld.eu/at/documentation/

This project is not statistic, there is no mechanism to make it learn words. Feel free to submit your words, you can use this Open office xcell document from which the database is imported:

https://github.com/xpheres/analyticaltranslator/tree/master/database

You can convert it to any other format if you want.

Anything else I can help with please just let me know!

**Comment by SparkyCola** [comment]
Thanks for your answer. What about using existing databases like wikitionary?

**Comment by xpheres** [comment]
by the way, I checked the words you told me and they do work:
Results of regen:
Translation (Spanish): 
lluvia
Results of tag:
Translation (Spanish): 
día

Please explain me the issue so I can address it.

**Comment by SparkyCola** [comment]
There are 32k german and 29k spanish words on wiktionary (that have a german/spanish translation).

**Comment by xpheres** [comment]
Yes we can use external databases but take into account that the database for this software needs more than just translation, you have to make sure that the following fields for every word are filled, otherwise the software can not recognize and correct grammar:

category (substantive, verb...)
gender (German gender)
word	case (German case)
spanish gender
spanish (Spanish translation)
updated (A field created to check new vocabulary)
english (English translation)

**Comment by xpheres** [comment]
If you know a way to import such a huge vocabulary and fill the fields automatically please let me know, I have no time to do it by hand but if there is a way to automate the process I will be happy to import such a huge vocabulary database.

**Comment by SparkyCola** [comment]
Well the QuickDic app from the F-Droid store can download these into a database which is saved as *.quickdic maybe i can find out what kind of database it is (you could check yhe sourcecode for that)

**Comment by SparkyCola** [comment]
You can get a dump of the German and Spanish Wiktionary [here](https://dumps.wikimedia.org/backup-index.html). Then you'll just have to write a script/program that can filter them (remove all where no translation exists and so on. I also figured out that [the MW Dumper](https://www.mediawiki.org/wiki/Manual:MWDumper) could help you with that.

**Comment by SparkyCola** [comment]
I didn't get a result for "Tag" in the JS-frontend, PHP seems to work.

**Comment by xpheres** [comment]
Thanks for leting me know, I think I found the reason, the reason is that the word in spanish have a written accent, it happens with every word that have a written accent, it seems that the JSON parser does not process any accent... thanks for leting me know, I didn't know... I will try to fix it later

**Comment by SparkyCola** [comment]
You're welcome
